web-ui/src/components/menu/Menu.jsx

```diff
@@ -147,36 +156,36 @@ function Menu() {
   const [selectedFile, setSelectedFile] = useState(null);
   const feedbackLinks = getFeedbackLinks(isAdmin, isPDL, isSupervisor);
 
-    const getReportLinks = () => {
-        const links = []
-
-        if (selectHasBirthdayAnniversaryReportPermission(state)) {
-            links.push(["/birthday-anniversary-reports", "Birthdays & Anniversaries"]);
```

Comment:
This link was replaced by two links below, one for Birthdays and one for Anniversaries.